Unable to compile jaxlib from source

I have been trying to compile jaxlib from source, and am unable to do so due to the following error:

```
bazel-out/k8-opt/bin/external/org_tensorflow/tensorflow/compiler/xla/python/pywrap_xla.cc:6626:64: error: cannot convert 'google::protobuf::RepeatedField<long int>*' to 'google::protobuf::RepeatedField<long long int>*' for argument '3' to 'bool xla::swig::HandleRepeatedInt64Attribute(PyObject*, const char*, google::protobuf::RepeatedField<long long int>*)'
         dimension_numbers4.mutable_lhs_contracting_dimensions())) {
                                                                ^
```

This seems to be an issue with TensorFlow.

Steps to reproduce the build failure:
1. `git clone https://github.com/google/jax`
2. `python build/build.py`

I have cleared my Bazel cache by performing `rm -rf ~/.cache/bazel`, to ensure that the cached version is not used.

Python: Miniconda3
MKLDNN enabled: Yes
CUDA enabled: No

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -17,10 +17,10 @@ http_archive(
 #    and update the sha256 with the result.
 http_archive(
    name = "org_tensorflow",
-   sha256 = "20f26f90359f7dad7472ebcb3142c5b6166e057ed89442614f9077c9b8d56278",
-   strip_prefix = "tensorflow-0ce305b6ce8560945be3a92eae1bcddd60af5e10",
+   sha256 = "37138f72ba6db3fcb9b805a8b3432831771be9c0a70f6fded4c12c392c6205e7",
+   strip_prefix = "tensorflow-9ad0810fd55096fc86a58300c5a2710b2f3b5175",
    urls = [
-       "https://github.com/tensorflow/tensorflow/archive/0ce305b6ce8560945be3a92eae1bcddd60af5e10.tar.gz",
+       "https://github.com/tensorflow/tensorflow/archive/9ad0810fd55096fc86a58300c5a2710b2f3b5175.tar.gz",
    ],
 )
 

